applications/OptimizationApplication/tests/algorithm_tests/test_line_search.py

```diff
@@ -66,6 +71,61 @@ def test_ConstantLineSearchNoneNorm(self):
         alpha = line_search.ComputeStep()
         self.assertEqual(alpha, 3.0)
 
+    def test_BBStepInfNorm(self):
```

Comment:
a test for the default params case is missing

Reply:
added